extension/src/popup/views/MnemonicPhrase.tsx

```diff
@@ -55,5 +55,11 @@ export const MnemonicPhrase = ({
     );
   }
 
-  return null;
+  return (
```

Comment:
Just want to confirm - what happens if you create an account, close the window at this stage, come back (are redirected to ` ROUTES.accountCreator` as stated here) and then create a new account instead of importing your old phrase?

Kind of an edge case, but just want to make sure that if a user does something weird, it doesn't put their account data into a weird state. I _think_ it should work fine and ignore the old keystore, but not 100% sure

Reply:
yeah so I think you're right that it does take you back to the create a password step if this happens. I think maybe we could also handle this by doing this instead - 
instead of routing back to account creator, we could add a new branch to that component that shows a password field and does a `SHOW_BACKUP_PHRASE` after collecting it.

My original thoughts were that at this stage if they abandon and come back, they has no way of key being used yet so easy enough to add this fallback but I could see how it would be better to do what I listed above. It is an edge case but maybe worth switching to getting the existing phrase?

Reply:
Yeah, I think maybe the most complete solution here is to have the user enter password and get back to `SHOW_BACKUP_PHRASE` 👍

Reply:
@piyalbasu I added this in [7c276d2](https://github.com/stellar/freighter/pull/1250/commits/7c276d281ddc8d5bab71d4873cd8528f8bf5eb61)